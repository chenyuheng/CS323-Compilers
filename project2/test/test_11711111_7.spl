struct Node {
    int key;
    int value;
    struct Node next;
};

int main(){
    struct Node node;
    node.key = 0;
    node.value = 2;
    node.key + 1 = 2;
}
