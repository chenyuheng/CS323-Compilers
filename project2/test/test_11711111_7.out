Error type 6 at Line 11: left side in assignment is rvalue
